dev/a11y_assessments/lib/use_cases/use_cases.dart

```diff
@@ -28,7 +29,14 @@ import 'text_field_password.dart';
 abstract class UseCase {
   String get name;
   String get route;
-  Widget build(BuildContext context);
+  Widget get mainWidget;
```

Comment:
why change this to a widget instead of a builder?

you can have a wrapper like this
```dart
Widget myActualBuild(BuildContext context) {
      return DynamicTitle(
      title: name,
      child: build(context),
    );
}
```